typescript/packages/common-memory/settings.ts

```diff
@@ -9,4 +10,14 @@ export const ttl = 60 * 60;
  * Default clock uses 10sec accuracy so that it's more stable
  * in CI environments.
  */
-export const clock = Clock.default.with(10)
+export const clock = Clock.default.with(10);
+
+/**
+ * Rate limiting configuration
+ */
+export const rateLimiting: RateLimitingOptions = {
+  baseThreshold: 300,
+  requestLimit: 30,
+  backoffFactor: 50,
+  maxDebounceCount: 17,
```

Comment:
> increased limits so that the client has a chance to do its throttling first, does that sound ok @Gozala ?

@ellyxir can you please suggest specific numbers here, it would save us some roundtrips. Thanks

Reply:
You can do it by clicking this button

<img width="773" alt="Screenshot 2025-03-07 at 9 36 46 AM" src="https://github.com/user-attachments/assets/9638c976-2970-4ad6-8951-76f8c370eee2" />